<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 35, 45);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>760</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 191);
</string>
    </property>
    <property name="text">
     <string>Программа контроля синхронизации каталогов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>760</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 191);
</string>
    </property>
    <property name="text">
     <string>(для всех файлов сравниваются контрольные суммы по алгоритму MD5)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>840</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="text">
     <string>исходный каталог</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>840</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>синхронизированный каталог</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>240</y>
      <width>640</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Проверить идентичность содержимого каталогов
и записать результат в лог (.txt)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Остановить
проверку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>logo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>841</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 191);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_File_1"/>
    <addaction name="action_File_2"/>
    <addaction name="action_File_3"/>
    <addaction name="separator"/>
    <addaction name="action_File_4"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_Help_1"/>
    <addaction name="action_Help_2"/>
    <addaction name="action_Help_3"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_File_1">
   <property name="text">
    <string>Выбрать исходный каталог</string>
   </property>
  </action>
  <action name="action_File_2">
   <property name="text">
    <string>Выбрать синхронизированный каталог</string>
   </property>
  </action>
  <action name="action_File_3">
   <property name="text">
    <string>Проверить идентичность содержимого каталогов</string>
   </property>
  </action>
  <action name="action_File_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Help_1">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_Help_2">
   <property name="text">
    <string>О резервном копировании методом синхронизации</string>
   </property>
  </action>
  <action name="action_Help_3">
   <property name="text">
    <string>Об односторонней идентичности содержимого каталогов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
